kratos/includes/define.h

```diff
@@ -93,18 +93,20 @@ catch(Exception& e) { Block throw Exception(e) << KRATOS_CODE_LOCATION << MoreIn
 catch(std::exception& e) { Block KRATOS_THROW_ERROR(std::runtime_error, e.what(), MoreInfo) } \
 catch(...) { Block KRATOS_THROW_ERROR(std::runtime_error, "Unknown error", MoreInfo) }
 
-#define KRATOS_CATCH_BLOCK_BEGIN class ExceptionBlock{public: void operator()(void){
-#define KRATOS_CATCH_BLOCK_END }} exception_block; exception_block();
-
-#ifndef __SUNPRO_CC
-#define KRATOS_TRY try {
-
-#define KRATOS_CATCH(MoreInfo) \
-  KRATOS_CATCH_WITH_BLOCK(MoreInfo,{})
+#ifdef KRATOS_DISABLE_CATCH
+    #define KRATOS_TRY { };
+    #define KRATOS_CATCH(MoreInfo) { };
 #else
-#define KRATOS_TRY { };
+    #ifndef __SUNPRO_CC
```

Comment:
do we still need this one?

Reply:
I doubt it, @pooyan-dadvand ?

Reply:
We don't need it. I don't even remember why I added it on its time...